core-plugins/src/test/java/org/apache/hadoop/fs/s3native/S3NInMemoryFileSystem.java

```diff
@@ -1,28 +0,0 @@
-/*
- * Copyright © 2015 Cask Data, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
- * use this file except in compliance with the License. You may obtain a copy of
- * the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations under
- * the License.
- */
-
-package org.apache.hadoop.fs.s3native;
```

Comment:
Is this unused?

Reply:
It says "remove after upgrading to hadoop 2.6.0". This is what we just did :)